variables.tf

```diff
@@ -100,6 +100,16 @@ variable "mirror_repositories" {
       description = "Mage-OS mirror fork of https://github.com/magento/zf1."
       teams       = ["continuous-integration"]
     }
+
+    mirror-mirror-commerce-admin.en = {
+      description = "Mage-OS mirror of https://github.com/AdobeDocs/commerce-admin.en."
+      teams       = ["continuous-integration"]
+    }
+
+    mirror-commerce-operations.en = {
+      description = "Mage-OS mirror of https://github.com/AdobeDocs/commerce-operations.en."
+      teams       = ["continuous-integration"]
+    }
```

Comment:
I wonder if the dot `.` in the names is okay...?

Reply:
They are not. From the terraform check:

    │ On variables.tf line 104: If this expression is intended to be a reference,
    │ wrap it in parentheses. If it's instead intended as a literal name
    │ containing periods, wrap it in quotes to create a string literal.

Reply:
I am unable to update the PR because of the branch protection rules. I guess I have to fork the repo.

Reply:
To ease things, I updated the branch.